README.md

```diff
@@ -1 +1,61 @@
-# math-magicians-
\ No newline at end of file
+# Math Magicians
+
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Built With
+
+- React (CRA)
+
+## Milestones
+
+- [✔️] Milestone 1: **Project setup**
+- [✔️] Milestone 2: **Components**
+- [✔️] Milestone 3: **Events**
+- [x] Milestone 4: **Full website**
+- [ ] Milestone 5: **Tests**
+- [ ] Milestone 6: **Deploy**
+
+## sneak peek
+
+![Application screenshot](src/calculator.png)
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove all these sections as this is not needed in regards to following and maintaining the standard in the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) structure to make your readme consistent and professional.